plugins/com.google.cloud.tools.eclipse.appengine.deploy/plugin.xml

```diff
@@ -43,4 +43,28 @@
     </handler>
   </extension>
 
+  <extension
+    point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https://.+\.appspot\.com">
```

Comment:
This seems broadly useful beyond the deploy console but includes the debugging console (e.g., the output from `dev_appserver` with URL links).  I suppose the `.localserver.ui` bundle could define its own pattern match listener and use the same class.